vsphere/datadog_checks/vsphere/api.py

```diff
@@ -161,28 +185,34 @@ def get_infrastructure(self):
 
             # Collect the objects and their properties
             res = content.propertyCollector.RetrievePropertiesEx([filter_spec], retr_opts)
-            mors = res.objects
+            obj_content_list = res.objects
             # Results can be paginated
             while res.token is not None:
                 res = content.propertyCollector.ContinueRetrievePropertiesEx(res.token)
-                mors.extend(res.objects)
+                obj_content_list.extend(res.objects)
         finally:
             view_ref.Destroy()
 
-        infrastructure_data = {mor.obj: {prop.name: prop.val for prop in mor.propSet} for mor in mors if mor.propSet}
+        infrastructure_data = {
+            obj_content.obj: {prop.name: prop.val for prop in obj_content.propSet}
+            for obj_content in obj_content_list
```

Comment:
I'm not sure about the renaming from mors to `obj_content_list`.
`mor` is a term that appears everywhere in the vsphere integration. I think it's more understandable to use it when possible.

Reply:
Yes, it's a bit confusing, but I made the change on purpose to better distinguish the `ObjectContent` and `ManagedEntity` (aka mor)

Here the `obj_content_list` is `List[PropertyCollector.ObjectContent]`.

But a mor refers to `vim.ManagedEntity`.

The `mor`/`vim.ManagedEntity` is actually `PropertyCollector.ObjectContent.obj`.

Reply:
From a typing perspective this is correct of course, but I still think `obj_content_list` and `obj_content` are very mysterious objects.
To please both aspects, mind adding an extra comment in that case? Simply stating that `obj_content` contains the reference to the ManagedEntity + its properties?

Reply:
Sure, just added a comment.